plugins/SitesManager/templates/siteWithoutData.twig

```diff
@@ -14,7 +14,8 @@
                 interval: 1000,
                 onUpdate: function () {
                     // reload page as soon as a visit was detected
-                    document.location.reload();
+                    var urlToday = broadcast.updateParamValue('date=today', window.location.href);
```

Comment:
does it make sense to use `propagateNewPage()` here? It will make sure the reload will be performed even if angular etc would maybe block it